rotkehlchen/globaldb/handler.py

```diff
@@ -240,23 +240,6 @@ def cleanup(self) -> None:
         if self._packaged_db_conn is not None:
             self._packaged_db_conn.close()
 
-    def _wal_checkpoint(self) -> None:
```

Comment:
Notice that this was no longer used @yabirgb

Reply:
Yes, oj removed all the usage